api/GL/glext.h

```diff
@@ -464,7 +464,6 @@ GLAPI void APIENTRY glBlendEquation (GLenum mode);
 
 #ifndef GL_VERSION_1_5
 #define GL_VERSION_1_5 1
-#include <KHR/khrplatform.h>
```

Comment:
This include is still needed. Else the next couple of lines miss the definition for the khronos_* types.

Reply:
Fixed